.phan/stubs/wpcom-stubs.php

```diff
@@ -1124,6 +1124,7 @@ class WPCOM_Features
         public const LEGACY_CONTACT = 'legacy-contact';
         public const LOCKED_MODE = 'locked-mode';
         public const MANAGE_PLUGINS = 'manage-plugins';
+        public const SCHEDULED_UPDATES = 'scheduled-updates';
```

Comment:
This addition would get overwritten tomorrow, which is why the file comment says this:

https://github.com/Automattic/jetpack/blob/404981f0dc26887856db802adafd5f4e4b2f89a0/.phan/stubs/wpcom-stubs.php#L5-L7

Since I'm looking at adding a different stub anyway, I'll add this too for you in D147296-code.

Reply:
Sorry I somehow missed that! Thanks so much for taking care of it @anomiex !